src/components/Search.tsx

```diff
@@ -1,16 +1,17 @@
 import React, { useState } from 'react';
 import { useHttpGet } from '../util/useHttp';
+import Box from '@material-ui/core/Box';
 
 type Data = {
   hits: [{ objectID: string; url: string; title: string }];
 };
 
-export default function Search(): JSX.Element {
+export default function SearchQuery(): JSX.Element {
```

Comment:
Oh, here is your search form. I guess a lot of naming gets confusing when you're just playing around and learning! Still, change the above from just 'Form' since that's too generic. Thanks.